backend/src/routes.js

```diff
@@ -1,14 +1,16 @@
 const express = require('express');
 const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const jwt_Mid = require('express-jwt');
```

Comment:
also here lets use a more JS-friendly name like 'jwtMiddleware' or something similar in camelcase.